tests/test_obs_controller.py

```
@@ -0,0 +1,118 @@
+    assert start_result.ok, f"Start recording failed: {start_result.error}"
+    
+    # Poll for recording to start (max 10 seconds)
+    max_wait = 10
+    start_time = time.time()
+    while time.time() - start_time < max_wait:
+        try:
+            rec_status = c.ws.get_record_status()
+            if hasattr(rec_status, 'outputActive') and rec_status.outputActive:
+                break
+        except Exception:
+            pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)